aikit/transformers/categories.py

```diff
@@ -477,3 +479,223 @@ def _get_model(self, X, y=None):
 
 
 # In[]
+
+class _OrdinalOneHotEncoder(BaseEstimator, TransformerMixin):
+
+    def __init__(self,
+                 categories="auto",
+                 dtype=np.int32
+                 ):
+        self.categories=categories
+        self.dtype=dtype
+        
+        
+    def get_feature_names(self, input_features=None):
+        
+        if input_features is None:
+            input_features = self._input_features
+            
+        if len(input_features) != self._nb_columns:
+            raise ValueError("input_features (length = (%d)) should have the same length has the number of columns (%d)" % (len(input_features), self._nb_columns))
+
+        features = []
+        for j, f in enumerate(input_features):
+            features += [str(f) + "_g_" + str(m) for m in self._all_inv_mapping[j].values[:-1]]
```

Comment:
I named the columns "{name_of_variable}_g_{name_of_modalities}".  'g' for 'greater'
It is not very good... 
* I could name them '{variable}>{modality}' .. but I don't really like including symbol like '>' in column name
* {variable}_greater_{modality} ... but maybe too long ?

any ideas/suggestions ?

Reply:
`g` or `gt` (greater than) may be more explicit, but still don't have a better suggestion 😉

Reply:
ok.. let's keep with 'g' then.